src/js/jsx/sections/style/StrokeList.jsx

```diff
@@ -117,7 +118,11 @@ define(function (require, exports, module) {
                     this.props.index,
                     blendMode,
                     LayerEffect.STROKE);
-            headlights.logEvent("edit", "stroke-blendmode-input", blendMode);
+
+            if (!this._logEventDebounced) {
+                this._logEventDebounced = synchronization.debounce(headlights.logEvent, null, 1000);
```

Comment:
It might actually be better to make the timeout on this larger? How long does it take you to browse through the menu to find your next blend mode... Maybe 10 seconds?